.env.local

```diff
@@ -0,0 +1,4 @@
+NEXT_PUBLIC_API_URL=https://api.dev.app.astrodao.com
```

Comment:
This file should not be present in the repo. Please remove this. You can provide proper description in README.md if you feel it's necessary.

Reply:
Is this a sensitive URL?

Reply:
And yes, it's absolutely necessary to have a working readme.

Reply:
It's not sensitive, just .env.local should not be part of the project. Moving variables to .env. an example is a better idea :)